.github/copilot-instructions.md

```diff
@@ -0,0 +1,91 @@
+# Copilot Instructions for Sous Chefs Cookbooks
+
+## Repository Overview
+
+**Chef cookbook** for managing software installation and configuration. Part of the Sous Chefs cookbook ecosystem.
+
+**Key Facts:** Ruby-based, Chef >= 16 required, supports various OS platforms (check metadata.rb, kitchen.yml and .github/workflows/ci.yml for which platforms to specifically test)
+
+## Project Structure
+
+**Critical Paths:**
+- `recipes/` - Chef recipes for cookbook functionality (if this is a recipe-driven cookbook)
+- `resources/` - Custom Chef resources with properties and actions (if this is a resource-driven cookbook)
+- `spec/` - ChefSpec unit tests
+- `test/integration/` - InSpec integration tests (tests all platforms supported)
+- `attributes/`, `libraries/`, `templates/` - Configuration, helpers, ERB templates
```

Comment:
Split these into individual bullets:
- attributes/ - Configuration for recipe driven cookbooks (not applicable to resource cookbooks)
- libraries/  - Library helpers to assist with the cookbook. May contain multiple files depending on complexity of the cookbook.
- templates/ - ERB templates that may be used in the cookbook
- files/ - files that may be used in the cookbook